app/i18n/es/people.ts

```diff
@@ -173,6 +188,7 @@ const people: People = {
           'CEO - The Brand Garden',
           `Master's Organizational Leadership - Henley`,
         ],
+        link: '',
```

Comment:
Link missing? https://www.linkedin.com/in/richardnall/

Reply:
👍🏼